docs/src/main/asciidoc/_elytron/components/Filesystem_Security_Realm.adoc

```diff
@@ -323,4 +324,24 @@ realm-name:realm-two
 
 There are other options that can be specified for converting a filesystem realm, the details of which can be found using the `./bin/elytron-tool.sh filesystem-realm-integrity -h` command. 
 
+== Updating Elytron-tool Generated CLI Scripts for Domain Mode
+As mentioned above, some of the features added to `elytron-tool` creates a CLI script that can be used to add a
+key-store resource and a filesystem-realm resource in the Elytron subsystem that makes use of the newly signed realm
+content. Some examples of these cli scripts include:
+
+. the <FILESYSTEM-REALM-NAME>.cli script generated when `filesystem-realm-encrypt` is used with the elytron-tool to encrypt an existing file-System realm.
+. the <FILESYSTEM-REALM-NAME>.cli script generated when `filesystem-realm-inregrity` is used with the elytron-tool to add integrity checking to existing filesystem realms.
+. the <FILESYSTEM-REALM-NAME>.sh script generated when `filesystem-realm` option is used with the elytron-tool to create a Filesystem realm from a legacy properties file.
+
+However, that cli script can only be used directly if the server is running in Standalone mode. For Domain mode
+the script needs to be updated. The steps for updating the script are as follows:
+
+* Update the cli file contents to include/add a /profile=<PROFILE_NAME> in front of all occurrences of the
+/subsystem=... strings as found in the file (so in front of /subsystem=keycloak... , /subsystem=elytron... ,
+/subsystem=undertow... ); where <PROFILE_NAME> is the profile name that you are using in your WildFly domain.xml
```

Comment:
Please put a period after 'domain.xml'.  So the next bit is its own sentence.

Reply:
done!

Reply:
include/add -> maybe we could just use add?

Reply:
fixed both!